src/terra-dev-site/guide/Internationalization/InternationalizationIntro.a.guide.md

```diff
@@ -0,0 +1,85 @@
+
+**Note:** *This component should be your root component and wrap all components in your app. This is to help ensure that the entire application is within the same configured i18n context provided by [react-intl's `<IntlProvider />`](https://github.com/yahoo/react-intl/wiki/Components#intlprovider). The terra-base component provides an abstraction around react-intl's `<IntlProvider />` component which allows code splitting of translations per locale.*
+
+You can [read more about react-intl's API here](https://github.com/yahoo/react-intl/wiki/API).
+
+If your app doesn't have any custom translations, you can set up and use terra-base as follows:
+
+```jsx
+import Base from 'terra-base';
+import App from './App'; // Your custom app component
+
+// This value could be sent from the server as well
+const locale = (navigator.languages && navigator.languages[0])
+               || navigator.language
+               || navigator.userLanguage
+               || 'en';
+
+<Base locale={locale}>
+  <App />
+</Base>
+```
+
+This will load the `en` translations for any Terra UI components you use in your app.
+
+**Note:** *This requires running terra-toolkit's aggregate translation tool to fully work. See the terra-toolkit's aggregate translation section below for more info.*
+
+If your app has custom translations outside of the ones provided by Terra UI, you can set up and use terra-base as follows:
+
+```jsx
+import Base from 'terra-base';
+import App from './App'; // Your custom app component
+import appTranslations from './aggregated-translations/en.js'; // Your aggregated app translations
+
+// This value could be sent from the server as well
+const locale = (navigator.languages && navigator.languages[0])
+               || navigator.language
+               || navigator.userLanguage
+               || 'en';
+
+<Base locale={locale} customMessages={appTranslations}>
+  <App />
+</Base>
+```
+
+**Note:** *This requires running terra-toolkit's aggregate translation tool to fully work. See the terra-toolkit's aggregate translation section below for more info.*
+
+[More info on terra-base can be found on it's documentation page.](https://engineering.cerner.com/terra-ui/#/components/terra-base/base/base)
+
+
+# HTML Attributes Which Help With Internationalization
+You'll also need to set up the following HTML attributes that help with i18n. You'll want to add a `dir` attribute which denotes the reading direction of your app's content as well as a `lang` attribute which denotes the language of your app.
+
+```html
+<html dir="ltr" lang="en">
+```
+
+We recommend setting this in your jest setup file as well.
+
+```js
+const htmlTag = document.getElementsByTagName('html')[0];
```

Comment:
Above you suggest setting the lang, should we also be setting the lang here in our Jest setup?

Reply:
Hmm, that's a good question. I don't see much reason to use the `lang` attribute in Jest tests. Some of the main reasons we recommend setting it in the HTML document are that it helps screen readers and tools like Google translate parse the page accroding to the specified language. Those features aren't needed when doing Jest tests so I wouldn't bother adding it to your Jest test setup. We also don't provide any custom styling or fonts based on the `lang` attribute and I don't see us adding any in the foreseeable future.

More info on the lang attribute and why to use it can be found here: https://www.w3.org/International/questions/qa-lang-why.en